content/projects/nodejs/file-io/_index.md

```diff
@@ -1,8 +1,10 @@
 ---
 title: Node & File IO
 ready: true
-weight: 1
-pre: "<b>1: </b>"
+prerequisites:
+  hard: ["workshops/node", "topics/js-and-node-specific/introduction-to-node"]
```

Comment:
This "workshops/node" redirects you to this "topics/js-and-node-specific/introduction-to-node"
It might make sense to have one, unless the workshop label makes a difference

Reply:
We'll probably get rid of the workshop actually